src/gui/poses.js

```diff
@@ -23,7 +23,7 @@ const POSE_JOINTS = {
   LEFT_ANKLE: "left ankle",
   RIGHT_ANKLE: "right ankle",
 };
-exports.POSE_JOINTS = POSE_JOINTS;
+//exports.POSE_JOINTS = POSE_JOINTS;
```

Comment:
If none of these previously constants are used then they should all be deleted

Reply:
I actually removed this script because there was a little remaining functionality. See new commit.